libs/eo/landing-page/shell/src/lib/eo-landing-page-shell.component.ts

```diff
@@ -35,17 +36,336 @@ const selector = 'eo-landing-page-shell';
   selector,
   styles: [
     `
+      @use '@energinet-datahub/watt/utils' as watt;
+
       ${selector} {
+        position: relative;
         display: block;
+
+        .${selector}__wrapper {
+          position: relative;
+          width: 100%; // This is used for styling 100% of the available screen width, with i.eg a background color
+          &.wave {
```

Comment:
question: Why is this not a Modifier?
```suggestion
          .${selector}__wrapper--wave {
```
- [ ] Remove nesting
- [ ] Update class name in component template